src/Microsoft.Identity.Web/MicrosoftGraph/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using Microsoft.Graph;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Extensions methods on a MicrososoftAppCallingWebApiAuthenticationBuilder builder
```

Comment:
```suggestion
    /// Extensions methods on a MicrososoftIdentityAppCallingWebApiAuthenticationBuilder builder
```

Reply:
fixed in this [PR](https://github.com/AzureAD/microsoft-identity-web/pull/442)

---
In reply to: [469486283](https://github.com/AzureAD/microsoft-identity-web/pull/434#discussion_r469486283) [](ancestors = 469486283)